layouts/shortcodes/tabpane.html

```diff
@@ -11,9 +11,11 @@
   {{ end -}}
 {{ end -}}
 
-{{ with .Get "persistLang" -}}
-  {{ if ne ( printf "%T" . ) "bool" -}}
-    {{ errorf "Shortcode %q: boolean value expected for parameter %q, but got %s. Error position: %s" $.Name "persistLang" (printf "%T" .) $.Position  -}}
+{{ $_persistLang := .Get "persistLang" -}}
```

Comment:
You are reusing parameter `persistLang` here. However since we now persist the tab and not the language any more, this is a kind of a misnomer.
What about adding a new parameter `persistTab` which supersedes the currently ued parameter `persistLang`? For now we would leave parameter `persistLang` in place (maybe we issue a deprecation message warning, informing users that this parameter will be dropped eventually). This way, we could preserve backwards compatibility while offering the new more flexible solution proposed from you.

Reply:
Done, but using a parameter name of `persist`.